deprecations/README.md

```diff
@@ -39,6 +39,10 @@ The table columns have the following meanings:
 
 | Feature | Comments | Deprecated | Discontinued |
 | :-- | :-- | :-- | :-- |
+| `TaxIdentifier` in [Close bill](../operations/bills.md#close-bill) | Replaced by `AccountTaxIdentifier` | 16 Nov 2023 | 16 Nov 2024 |
+| `Address` in [Close bill](../operations/bills.md#close-bill) | Replaced by `AccountAddress` | 16 Nov 2023 | 16 Nov 2024 |
+| `CompanyId` in [Bill](../operations/bills.md#bill) | Replaced by `AssociatedAccountIds` | 16 Nov 2023 | 16 Nov 2024 |
+| `CompanyDetails` in [Bill](../operations/bills.md#get-all-bills) | Replaced by [AssociatedAccountData](../operations/bills.md#bill-associated-account-data) | 16 Nov 2023 | 16 Nov 2024 |
```

Comment:
I'd still mention `CompanyAddress` and `CompanyTaxIdentifier` here. Even though they're not mentioned in the docs, they exist in the code. But this is also more of a question, so @VBuben would love to hear your thoughts 😄